db/seeds.rb

```diff
@@ -18,6 +18,13 @@ def format_errors(model = nil)
     ForemanTasks.sync_task(::Actions::Katello::Organization::Create, org) unless org.library
     User.current = nil
   end
+  if Setting['db_pending_seed']
```

Comment:
the goal was to allow the user to set their 'default org' attribute back to nil after installation and not reset it to the pre-made org.  I think this is the only way to do that.